src/parsers/parseArgs.ts

```diff
@@ -95,17 +92,6 @@ export function parseArgs(str: string): [string[], string] {
       case "\n":
         pushArg();
         break loop;
-      case "~":
-        if (currentArg === "") substitute(process.env.HOME as string);
-        else pushChar(char);
-        break;
-      case ".":
-        if (str[i + 1] === "/") substitute(cwd);
-        else if (str[i + 1] === "." && str[i + 2] === "/") {
-          i++;
-          substitute(join(cwd, ".."));
-        } else pushChar(char);
-        break;
```

Comment:
These weren't just for `cd`, you know. They might've been useful for arg parsing anyway. We'll have to stop using shell: true at some point so this will be helpful then.

Reply:
this makes the cd output confusing
```sh
$ cd ./notadir
not a directory: /full/dir/to/notadir```

so this should just be passed as plain string and be parsed elsewhere